README.md

```diff
@@ -1,94 +1,79 @@
-This was the result of a discovery sprint, I am in the process of cleaning it up for easier local install.
-
 ## Local set up
 
-This is using Pipenv and Postgres
-
-### Pipenv info
-
- https://github.com/pypa/pipenv
-
-### Postgres
-
- https://www.postgresql.org/download/
-
-I recommend [pgcli](https://www.pgcli.com/) for interfacing with postgres
+Install Docker
 
+    https://www.docker.com/get-started
 
-In pgcli or your sql shell command to create datebase with:
 
-    create database crt_portal;
+Use a long, random string for your secret key and save it in your local environment.
 
+    export SECRET_KEY=''
 
-commands to create local db user for app, replace `username` and `userpass` with your choices:
+To build the project
 
-    CREATE USER username WITH PASSWORD 'userpass' CREATEDB;
-    GRANT permissions ON DATABASE crt_portal TO username;
+    docker-compose up -d --build
 
+To run the project
 
-(I would like the local set up to be in docker next. so we don't need to to the previous steps)
+    docker-compose u
 
-Once you crate the crt_portal postgres database, set the variables in your local environment accordingly.
 
-    export =''
-    export DB_USER=''
-    export DB_PASSWORD=''
-    export DB_HOST='local'
-    export SECRET_KEY=''
-    export ENV='LOCAL'
+create a superuser for admin access
 
-Add those variables to your bash_profile so that you don't have to set them going forward.
+     docker-compose run web python /code/crt_portal/manage.py createsuperuser
 
-The 'Local' setting will trigger the local_settings.py for easier setup.
 
-If you have pipenv installed and your database set up, you can run the following commands to get the project running:
+To add some test data after you log in at `http://0.0.0.0:8000/admin/login`
 
-    pipenv install
 
-    cd crt_portal
+in another terminal if you are doing front end work:
 
-apply the data migrations
+    gulp watch
 
-    python manage.py migrate
 
-create a superuser for admin access
 
-    python manage.py createsuperuser
+## cloud.gov set up
 
-run the server on localhost:8000
+- create postgres DB and S3 with development settings:
```

Comment:
@LindsayYoung Is the function/purpose of the S3 bucket to hold static assets like JS and CSS files? Could we add a note about that here?

Reply:
We talked abut this and documenting that we should add better explanations once we have cloud.gov running static assets in #9